### ISSUE TITLE
Android build failure

### ISSUE THREAD

**Issue by Wayward1** [minetest/minetest#5585]
The build fails with
```In file included from jni/../jni/src/script/clientscripting.h:24:0,
                 from jni/../jni/src/script/clientscripting.cpp:21:
jni/../jni/src/script/cpp_api/s_base.h:27:17: fatal error: lua.h: No such file or directory
 #include <lua.h>
                 ^
compilation terminated.
/home/home/android-ndk-r11c/build/core/build-binary.mk:462: recipe for target 'obj/local/armeabi-v7a/objs/minetest/jni/src/script/clientscripting.o' failed
make[2]: *** [obj/local/armeabi-v7a/objs/minetest/jni/src/script/clientscripting.o] Error 1```

**Comment by nerzhul** [comment]
I'm working on a fix, it seems we miss an include directory

**Comment by nerzhul** [comment]
Fixed by 98286a24bce14627295230fd3b846abf9641c086
